joomla/joomla-cms
# [4.3] After saving a menu item, rebuild the tree

### Summary of Changes
While working on the tags, I noticed that the menu system is missing a rebuild upon saving an item. We have this code in com_categories and in com_tags. Without this, errors in the tree are not fixed upon saving the item again. 


### Testing Instructions
In a website with a few menu items, change the lft and rgt value from a menu item to 0. Edit that menu item and save it again. See that the lft and rgt values are restored.

### Link to documentations
Please select:
